### ISSUE TITLE
[MU4 Issue] Crashing after cutting and pasting multiple measures

### ISSUE THREAD

**Issue by Mexifri98** [musescore/MuseScore#15002]
Cut and Pasting various measures crashes MU4

The expected behavior is for the program to cut and paste the measure I've selected without it crashing 

**Platform information**
 - OS: Windows 10 Version 22H2
 - Musescore 4 Version 4.0.0.223362132

**Additional context**

https://user-images.githubusercontent.com/119981399/206049977-06fc4989-bfd5-4551-b92a-3b2299b09ecb.mp4

**Comment by MarcSabatella** [comment]
Please attach the score itself so others can try to reproduce this.  You can either ZIP the file or else just rename the file to end in “.zip”, so that this site will accept the attachment.

**Comment by Mexifri98** [comment]
[Choir Example.zip](https://github.com/musescore/MuseScore/files/10172000/Choir.Example.zip)

**Comment by MarcSabatella** [comment]
FWIW, I can't reproduce, but also, the steps aren't entirely clear.  It looks like you've made some modifications between the score you posted and what you show in the video - the video is in continuous view, for example.  Also, it seems maybe you are copying into a separate file?  Best to list the precise steps to reproduce the problem, starting with:

1. Load the score
2. ???
3. ???

etc.

**Comment by Mexifri98** [comment]
Ok, this video starts off from when I open the score normally, all I want to do is add more measures and move the musical passage over so I can write an intro for the piece. 

1. Add one measure to the end of the piece
2. Cut and paste the last measure
3. Add ties and slurs where needed to the measure that was just pasted
4. Cut the musical passage
5. Paste the musical passage on whatever measure 
6. Then it crashes

[Choir Example Video.zip](https://github.com/musescore/MuseScore/files/10172306/Choir.Example.Video.zip)

**Comment by MarcSabatella** [comment]
Hmm, your video shows a lot of manipulation that's really hard to follow - looks like you inserted measures, then deleted some, then cut and paste?  Also it appears you changed to contiunuous view at some point.  Anyhow, I've tried a bunch of things that look similar but no luck reproducing the crash.  It would be useful it you wrote out precisely what you mean by "add ties and slurs", since somehow I think that's probably the key to the crash.  Also be sure it's really the same copy of the file.

Also, it might be Windows only; I'm tesitng on Linux.

**Comment by Mexifri98** [comment]
I've added commentary to the video to try and help explain what I'm doing or trying to do
[Choir Example.zip
[Choir Example video.zip](https://github.com/musescore/MuseScore/files/10172518/Choir.Example.video.zip)
](https://github.com/musescore/MuseScore/files/10172511/Choir.Example.zip)

**Comment by Tantacrul** [comment]
Putting this in Release, since @Eism has already found the fix (and the culprit). It's something that will cause general copy+paste problems every once in a while, so we think it's safe to do now.
